vmtkScripts/vmtkbranchpatching.py

```diff
@@ -38,7 +38,7 @@ def __init__(self):
 
         self.PatchSize = [0.0, 0.0]
 
-        self.GroupIdsArrayName = ''
+        self.GroupIdsArrayName = 'GroupIds'
         self.LongitudinalMappingArrayName = ''
         self.CircularMappingArrayName = ''
```

Comment:
These are the only ones missing as far as I can tell.

They should be set to `AbscissaMetric` and `AngularMetric`, respectively, by default. They are produced by `vmtkbranchmetrics`.